editor.planx.uk/src/@planx/components/FileUpload/Public.tsx

```diff
@@ -21,10 +21,11 @@ const FileUploadComponent: React.FC<FileUpload> = (props) => {
       props,
       slots.map((slot) => ({
         url: slot.url,
-        filename: slot.file.path,
+        filename: slot.file.name,
         cachedSlot: {
           ...slot,
           file: {
+            name: slot.file.name,
```

Comment:
This was a little unexpected, I've not exactly pinned down what's caused this change here. 

`path` is now returning an actual path (e.g. `./file.jpg`) whilst `name` is just the filename (e.g. `file.jpg`). There's a few associated test, mock, and UI changes related to this.